### PR TITLE
feat: Jump in item new flow

### PR DESCRIPTION
**Pull request by braianj**

### CODE REVIEW

File: src/components/Button/JumpInPosition.tsx [hunk 1]
```diff
@@ -19,55 +29,124 @@ export type JumpInPositionProps = React.HTMLProps<HTMLAnchorElement> & {
...
+        title: string
+        description: string
+        buttonLabel: string
+      }
+    | false
+  >(false)
+  const [servers] = useAsyncMemo(getReamls)
+
+  const isPosition = !!event
   const position = isPosition ? event && `${event.x},${event.y}` : "HTTP"
```
Comment (aleortega) [inline src/components/Button/JumpInPosition.tsx:48]: I know this is not related to the current change but it confuses a bit (worth to mention for future refactor maybe) but the doubt that it arises on me is: how a X,Y position can be related to the protocol HTTP?

File: src/components/Button/JumpInPosition.tsx [hunk 2]
```diff
@@ -19,55 +29,124 @@ export type JumpInPositionProps = React.HTMLProps<HTMLAnchorElement> & {
...
+      <ConfirmModal open={!!modalText} onClose={() => setModalText(false)}>
+        <div>
+          <img src={ExplorerJumpinImage} alt="Explorer Jump In" />
+        </div>
+        <Title>{modalText && modalText.title}</Title>
+        <Paragraph>{modalText && modalText.description}</Paragraph>
+        <Button
+          primary
+          onClick={handleModalClick}
+          style={{ marginTop: "28px" }}
```
Comment (aleortega) [inline src/components/Button/JumpInPosition.tsx:145]: expected in-line?

File: src/config/prod.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
   "GATSBY_CHAIN_ID": "1,137",
   "GATSBY_DECENTRALAND_URL": "https://play.decentraland.org",
+  "GATSBY_DECENTRALAND_DOWNLOAD_URL": "https://decentraland.org/download/",
```
Comment (aleortega) [inline src/config/prod.json:4]: Are these being used?

File: src/modules/servers.ts [hunk 4]
```diff
@@ -22,6 +23,23 @@ export const getServers = memo(
   { ttl: Infinity }
 )
 
+export const getReamls = memo(
+  async () => {
+    const servers: { address: string }[] = cache.catalysts.mainnet
```
Comment (aleortega) [inline src/modules/servers.ts:28]: always mainnet?

**Review by aleortega — APPROVED** [review]

File: src/components/Button/JumpInPosition.tsx (hunk 1 above)
Comment (braianj) [reply src/components/Button/JumpInPosition.tsx:48]: I can't.

File: src/components/Button/JumpInPosition.tsx (hunk 2 above)
Comment (braianj) [reply src/components/Button/JumpInPosition.tsx:145]: nice

File: src/config/prod.json (hunk 3 above)
Comment (braianj) [reply src/config/prod.json:4]: yes, there are used

**Pull request merged**